lib/beaker/local_connection.rb

```diff
@@ -53,9 +53,12 @@ def execute command, options = {}, stdout_callback = nil, stderr_callback = stdo
           result.stdout << std_out
           result.stderr << std_err
           result.exit_code = status.exitstatus
+          @logger.info(result.stdout)
+          @logger.info(result.stderr)
```

Comment:
Given this may be used and printed in other ways, isn't `debug` more appropriate?

Reply:
Well with a normal beaker run (beaker provisions a VM), the command output is printed using the default info log level.
So I think that we should keep the same behaviour when running on localhost.